### DISCUSSION TITLE
How to bind data to properties

### DISCUSSION THREAD

**Discussion by itcozum** [add2cal/add-to-calendar-button#505] [Q&A] [answered]
I'm using angular 17x and having issue binding data to properties
sample code is below
 <add-to-calendar-button name="[Reminder] Test the Add to Calendar Button"
                                        startDate="{{item.startDate | myDateFormat}}" 
                                        endDate="{{item.endDate | myDateFormat}}" 
                                        startTime="{{item.startDate | myTimeFormat}}" 
                                        endTime="{{item.endDate | myTimeFormat}}"
                                        location="World Wide Web" 
                                        description="item.description|sanitizeHtml"
                                        options="'Apple','Google','iCal','Outlook.com','Microsoft 365','Microsoft Teams','Yahoo'"
                                        identifier="{{item.id}}" lightMode="bodyScheme" #myCalendarButton>
                                    </add-to-calendar-button>

**Comment by jekuer** [comment, marked as answer]
Hi @itcozum,

Moves this into the discussions board as it is no "Issue".
Have a look at the stackblitz example at https://stackblitz.com/edit/add-to-calendar-button-angular?file=src%2Fapp%2Fapp.component.html - this should help 😃 .

**Reply by MaximeRobion** [reply]
@jekuer The stackblitz doesn't work with data binding, neither using [attr.name]="name" nor using name="{{name}}". Can you please edit it to make it work, or show a working example?

**Reply by jekuer** [reply]
Fixed with latest version. Stackblitz see https://stackblitz.com/edit/add-to-calendar-button-angular?file=src%2Fmain.ts
